src/mpi/coll/src/coll_impl.c

```diff
@@ -256,3 +256,27 @@ void MPIR_Coll_host_buffer_free(void *host_sendbuf, void *host_recvbuf)
     MPL_free(host_sendbuf);
```

Comment:
The other two functions are using `MPIR_` prefix. I think we should make it consistent, all use `MPIR_` or all use `MPII_`

Reply:
Done.